src/Sulu/Bundle/MarkupBundle/Tag/TagNotFoundException.php

```diff
@@ -16,33 +16,19 @@
  */
 class TagNotFoundException extends \Exception
 {
-    /**
-     * @var string
-     */
-    private $namespace;
-
-    /**
-     * @var string
-     */
-    private $tagName;
-
-    /**
-     * @var string
-     */
-    private $type;
-
     /**
      * @param string $namespace
      * @param string $tagName
      * @param int $type
```

Comment:
@mamazu not sure but I think this should be `string`: https://github.com/sulu/sulu/blob/08a40b7de9c1b3f9a6c42edc928ee8e4ff9ea97c/src/Sulu/Bundle/MarkupBundle/Tag/TagRegistryInterface.php#L23